wiki/Guides/Videos_from_YouTube/en.md

```diff
@@ -6,7 +6,7 @@ tags:
 
 # Videos from YouTube
 
-*Please note that downloading videos from YouTube is a breach of their Terms of Service.*
+*Please note that downloading videos from YouTube is a breach of their [Terms of Service](https://www.youtube.com/t/terms).*
 [YouTube](https://youtube.com) is a very useful resource for creators looking to add background videos to their [beatmaps](/wiki/Beatmap). If a song has a music video, it will most likely be on YouTube, and probably compressed at that golden point where the video looks decent but will not be too big in size.
```

Comment:
needs a blank line in between two paragraphs, here and in other translations

```suggestion
*Please note that downloading videos from YouTube is a breach of their [Terms of Service](https://www.youtube.com/t/terms).*

[YouTube](https://youtube.com) is a very useful resource for creators looking to add background videos to their [beatmaps](/wiki/Beatmap). If a song has a music video, it will most likely be on YouTube, and probably compressed at that golden point where the video looks decent but will not be too big in size.
```